dev-client/src/screens/ProjectInputScreen/ProjectInputScreen.tsx

```diff
@@ -127,19 +128,15 @@ export const ProjectInputScreen = ({
           <SoilPitSettings projectId={projectId} />
         </Accordion>
         <Box height={4} />
-        <RestrictByProjectRole role="MANAGER">
-          <Accordion
-            Head={
-              <Text pt={3} pb={3} fontSize="md" color="primary.contrast">
-                {t('soil.project_settings.required_data_title')}
-              </Text>
-            }>
-            <RequiredDataSettings
-              projectId={projectId}
-              enabled={allowEditing}
-            />
-          </Accordion>
-        </RestrictByProjectRole>
+        <Accordion
+          initiallyOpen={true}
+          Head={
+            <Text pt={3} pb={3} fontSize="md" color="primary.contrast">
+              {t('soil.project_settings.required_data_title')}
+            </Text>
+          }>
+          <RequiredDataSettings projectId={projectId} enabled={allowEditing} />
+        </Accordion>
```

Comment:
Suggestion: add a bit of space below the accordion

I believe that is part of the task of https://github.com/techmatters/terraso-mobile-client/issues/956